### PR TITLE
Community PR - Dune Dashboard 

### PR DESCRIPTION
**Pull request by mkchungs**
# Community PR - Dune Dashboard

Following the [Dune Integration announcement](https://polkadot.network/blog/polkadot-kusama-analytics-dune), Colorforful Notion has made significant progress in onboarding Polkadot, Kusama, and many parachains, and has built a few exemplary dashboards on [Dune](https://dune.com/substrate).

This Community PR includes documentation, useful tips in writing dune queries, and analytic dashboards for Polkadot ecosystem chains that we have onboarded during phase 1:

Screenshot 2024-05-31 at 3 12 24 PM

We will submit two more PRs (phase 2a and phase 2b) for 20+ additional parachains dashboards we have built in the following weeks

**Documentation Previews**

Polkadot-Dashboards-Overview-·-Polkadot-Wiki

### CODE REVIEW

**Review by DrW3RK — COMMENTED** [review]
Thank you! We will review it early this week.

File: docs/general/dune-analytics/parachain-dashboards/assethub-dashboards.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+---
+id: assethub-dashboards
+title: AssetHub Dashboards
```
Comment (profradha) [inline docs/general/dune-analytics/parachain-dashboards/assethub-dashboards.md:3]:
```suggestion
title: Asset Hub Dashboards
```

File: docs/general/dune-analytics/parachain-dashboards/assethub-dashboards.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+sidebar_label: AssetHub
```
Comment (profradha) [inline docs/general/dune-analytics/parachain-dashboards/assethub-dashboards.md:4]:
```suggestion
sidebar_label: Asset Hub
```

File: docs/general/dune-analytics/parachain-dashboards/assethub-dashboards.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+description:
+  AssetHub is a specialized parachain on Polkadot designed to facilitate the handling and management
```
Comment (profradha) [inline docs/general/dune-analytics/parachain-dashboards/assethub-dashboards.md:6]:
```suggestion
  Asset Hub is a specialized parachain on Polkadot designed to facilitate the handling and management
```

File: docs/general/dune-analytics/parachain-dashboards/assethub-dashboards.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+  of digital assets across various applications. It supports asset tokenization, trading, and
+  bridging services within the Polkadot ecosystem.
+keywords: [polkadot, dashboard, dune, assethub, digital assets]
+slug: ../assethub-dashboards
+---
+
+# AssetHub Dashboards
```
Comment (profradha) [inline docs/general/dune-analytics/parachain-dashboards/assethub-dashboards.md:13]:
```suggestion
# Asset Hub Dashboards
```

File: docs/general/dune-analytics/parachain-dashboards/assethub-dashboards.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+
+## Overview
+
+AssetHub is a specialized parachain on Polkadot designed to facilitate the handling and management
```
Comment (profradha) [inline docs/general/dune-analytics/parachain-dashboards/assethub-dashboards.md:17]:
```suggestion
Asset Hub is a specialized parachain on Polkadot designed to facilitate the handling and management
```

File: docs/general/dune-analytics/parachain-dashboards/assethub-dashboards.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+of digital assets across various applications. It supports asset tokenization, trading, and bridging
+services within the Polkadot ecosystem.
+
+## Featured Dashboards on Dune
+
+Here you'll find a variety of dashboards that help visualize data from the AssetHub parachain:
```
Comment (profradha) [inline docs/general/dune-analytics/parachain-dashboards/assethub-dashboards.md:23]:
```suggestion
Here you'll find a variety of dashboards that help visualize data from the Asset Hub parachain:
```

File: docs/general/dune-analytics/parachain-dashboards/assethub-dashboards.md [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+
+- [AssetHub](https://dune.com/substrate/assethub): Explore comprehensive analytics on digital asset
```
Comment (profradha) [inline docs/general/dune-analytics/parachain-dashboards/assethub-dashboards.md:25]:
```suggestion
- [Asset Hub](https://dune.com/substrate/assethub): Explore comprehensive analytics on digital asset
```

File: docs/general/dune-analytics/parachain-dashboards/assethub-dashboards.md [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+  management and operations within the AssetHub parachain.
+
+## Key Tables
+
+Data from the AssetHub parachain is organized into several key tables: `assethub.balances`,
```
Comment (profradha) [inline docs/general/dune-analytics/parachain-dashboards/assethub-dashboards.md:30]:
```suggestion
Data from the Asset Hub parachain is organized into several key tables: `assethub.balances`,
```

File: docs/general/dune-analytics/parachain-dashboards/assethub-dashboards.md [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+`assethub.blocks`, `assethub.calls`, `assethub.events`, `assethub.extrinsics`, `assethub.transfers`
+
+## Useful Queries
+
+Here are **materialized** or **parameterized** queries for Unique that may be useful to build your
+own charts:
+
+- [AssetHub Sum of Stablecoin on Each Chain by Time](https://dune.com/queries/3526956)
```
Comment (profradha) [inline docs/general/dune-analytics/parachain-dashboards/assethub-dashboards.md:38]:
```suggestion
- [Asset Hub Sum of Stablecoin on Each Chain by Time](https://dune.com/queries/3526956)
```

**Review by DrW3RK — APPROVED** [review]

**Pull request merged**